WalletWasabi.Fluent/Styles/Button.axaml

```diff
@@ -18,4 +24,28 @@
   <Style Selector="Button.action:disabled /template/ ContentPresenter">
     <Setter Property="Background" Value="{StaticResource ActionButtonDisabledBackgroundColor}" />
   </Style>
-</Styles>
\ No newline at end of file
+
+
+  <Style Selector="Button.plain">
+    <Setter Property="VerticalAlignment" Value="Stretch" />
+    <Setter Property="HorizontalAlignment" Value="Stretch" />
+    <Setter Property="Cursor" Value="Hand" />
+    <Setter Property="Padding" Value="0" />
+    <Setter Property="BorderThickness" Value="0" />
+  </Style>
+
+  <Style Selector="Button.plain /template/ ContentPresenter">
+    <Setter Property="Background" Value="{DynamicResource InvisibleButtonBackgroundColor}" />
+  </Style>
+
+
+  <Style Selector="Button.function">
+    <Setter Property="Padding" Value="14 10" />
+    <Setter Property="Background" Value="Transparent" />
```

Comment:
Need to add a background to make them more noticeable.

Reply:
we went through a few options and decided on background to the header.